src/plugins/jni/README.md

```diff
@@ -47,6 +48,25 @@ and run:
 
     sudo ldconfig
 
+### Java prerequisites on macOS ###
+
+macOS includes an old apple specific version of java, based on 1.6. However, for the jni plugin version 1.8 of Java is required, so either the openjdk or the oracle jdk has to be installed.
```

Comment:
i haven't tried as java 1.6 is not bundled with macOS anymore since 10.11, and as seen on https://www.java.com/en/download/faq/java_mac.xml#yosemitejava they don't support it anymore and recommend to use newer java versions by oracle instead:

> Apple has posted notice that Mac OS X 10.11 (El Capitan) will be the last OS X release that supports Java 6, and as such, recommends developers whose applications depend on Java 6 to migrate their apps to a newer Java version provided by Oracle.

should i try it anyway if it would work?